octoprint_mrbeam/templates/settings/debug_settings.jinja2

```diff
@@ -18,9 +18,9 @@
             <div class="controls">
                 <label class="checkbox">
                     <input type="checkbox" data-bind="checked: settings.plugins.mrbeam.vorlon">
-                    <strong>Enable Vorlon</strong>
-                    <div>Requires full page reload: <strong>Shift + F5</strong> (Windows and Linux) or <strong>Ctrl + Shift + r</strong> (Mac)</div>
-                    <div>Experimental Debug Service. Enable only if Mr Beam support team asked you to do so. If enabled it'll send a huge amount of browser debugging data to a Mr Beam server over an unsecured connection. Disables automatically after 6 hours.</div>
+                    <strong>{{ _('Enable Vorlon') }}</strong>
```

Comment:
remove this vorlon stuff, please

Reply:
Do you mean to remove the whole Vorlon block of code?

Reply:
yes, removed